### ISSUE TITLE
How to install in 2016

### ISSUE THREAD

**Issue by Kezzsim** [bklaas/fontreplacer#1]
<Update> The repository for this goes down from time to time, I was impatient when I thought I couldn't find a solution, but after about 24 hours the installer for this started showing up in the Applet Installer under Advanced on the Squeezebox Radio. Fantastic plugin, thanks again for writing it!
Saves a ton of time if you have more than one radio~

**Comment by bklaas** [comment]
Kezzsim-- did you figure this out?

My suggestion was going to be to scp the file to the squeezebox, which will work except I needed to review where the file goes on the squeezebox filesystem. I'd like to capture what you did to make this work so I can help anyone else that comes along with the same problem. Cheers.

**Comment by Kezzsim** [comment]
I believe I was able to get it working by linking directly to the xml repo that you have hosted here on github which I added to my LMS's third party plugin path (at the bottom of the plugins page)

Of course, Github uses HTTPS so I had to install `sudo pacman -S perl-io-socket-ssl` 
(pacman because I'm on arch).

Now I can get it to show up just fine in the applet installer so I don't need to manually scp it over. 

Thanks!
